manpages/onion-crl.md

```diff
@@ -0,0 +1,48 @@
+# ONION-CRL 1
+# NAME
+onion-crl -- Onion Compile Run Loop watches for source changes to automatically recompile and rerun.
+
+# SYNOPSYS
+onion-crl *./my-onion-server*
+
+# DESCRIPTION
+onion-crl inspects the passed executable to find out from which sources it was
+generated. It then watches the source files, and if any has changed, reruns the
+make file, kills the executable, and re runs it.
+
+On normal servers, as state is stored into a database making the executable
+stateless, this effectively creates a live C programming environment where the
+programmer changes some C or HTML code and reloads the page to see the changes.
+
+If the server is stateful, onion-crl is not recommended.
+
+onion-crl assumes a cmake environment where its running into a `build` directory,
+separated from the source, and with the `make` as compilation program.
+
+# ENVIRONMENT
+onin-crl tries its best to find all source files, but if its not working
```

Comment:
Suggested wording: onion-crl tries to find all the source files, but if unsuccessful these environment variables can affect the behavior